runtime/tests/mock.rs

```diff
@@ -15,7 +15,6 @@
 // along with CENNZnet.  If not, see <http://www.gnu.org/licenses/>.
 
 #![allow(dead_code)]
-#![cfg(test)]
```

Comment:
I thought we needed this attribute to include it in cargo test

Reply:
It's not neccsesary for integration tests since they're in `/tests/` folder which cargo knows about